src/components/Board/Symbol/Symbol.js

```diff
@@ -106,7 +128,18 @@ function Symbol(props) {
         )}
       {src && (
         <div className="Symbol__image-container">
-          <img className="Symbol__image" src={src} alt="" />
+          {isLoading && (
+            <div className="Symbol__image-loading">
+              <CircularProgress size={24} />
```

Comment:
We need to find a more subtle indicator for the loading state, because having a CircularProgress in each tile causes too many movements in the board. I think is better to leave a space until the loading. 
We can use only the display 'none' to avoid the display of the old src